Corona-Warn-App/src/main/res/layout/view_consent_status.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you use a ConstraintLayout to flatten the view hierarchy?

Reply:
Done ✅